bot/cogs/cyber.py

```diff
@@ -34,21 +35,21 @@ def contains_words(self, text: str, all_words: Iterable[str], *any_words: Iterab
         """
 
         # Gather HQ data from CyberStart Game.
-        with open("headquarters.txt", "r") as f:
-            game_docs = [level.split(":::\n") for level in f.read().split(";;;;;;\n")]
+        with open("headquarters.json", "r") as f:
+            game_docs = load(f)
 
-        if level_num not in range(len(game_docs)):
+        if level_num not in range(len(game_docs) + 1):
             await ctx.send("Invalid level number!")
 
-        elif challenge_num not in range(len(game_docs[level_num])):
+        elif challenge_num not in range(len(game_docs["L" + str(level_num)]) + 1):
```

Comment:
You should use f-strings as an equivalent for `str.format`. The f-string equivalent of this would be:
```py
elif challenge_num not in range(len(game_docs[f"L{level_num}"]) + 1):
```